molecule/command/base.py

```diff
@@ -32,7 +32,7 @@
 from molecule import util
 
 LOG = logger.get_logger(__name__)
-MOLECULE_GLOB = 'molecule/*/molecule.yml'
+MOLECULE_GLOB = os.environ.get('MOLECULE_GLOB', 'molecule/*/molecule.yml')
```

Comment:
Do we also need to register this in `molecule/config.py`? Can we also get a test for it?

Reply:
@lwm I do not know how to add a test for this right now as I have problems even running the test suite locally.

If you know how to add it please feel free to update it.

My personaly take on this would be to introduce it like this (as an undocumented feature) so we  can have time to work on a more complex usage scenario which would:

- add test suite for multiple roles
- document multiple role usage
- add a working example of multi roles (may be the same used for testing it)